test/partiql-randomized-tests/src/test/kotlin/org/partiql/lang/randomized/syntax/PartiQLParserDateTimeRandomizedTests.kt

```diff
@@ -53,7 +52,6 @@ class PartiQLParserDateTimeRandomizedTests {
     }
 
     @Test
-    @Disabled("The planner does not return the v1 plans right now. See assertExpression.") // TODO
```

Comment:
(self-review) these tests passed when re-enabled.

There is another file that has some datetime evaluation tests -- https://github.com/partiql/partiql-lang-kotlin/blob/enable-some-randomized-tests/test/partiql-randomized-tests/src/test/kotlin/org/partiql/lang/randomized/eval/EvaluatingCompilerDateTimeRandomizedTests.kt. Those tests depend on the old datetime modeling. Once that modeling is updated in https://github.com/partiql/partiql-lang-kotlin/pull/1691, we can try re-enabling those tests.